### PR TITLE
Revise warning copy when adding custom network with existing currency symbol

### PR DESCRIPTION
**Pull request by vthomas13**
## **Description**
This PR addresses the issue of users being scammed by being coached to add custom testnet or worthless tokens with well-known cryptocurrency symbols. The PR updates a warning message copy for a mismatch between the Chain ID and the Currency Symbol

## **Manual testing steps**

1. Run `yarn start` and open extension
2. Go to Settings > Network > Add New Network
3. Add Chain Id 137 (polygon) and Currency Symbol ETH
4. Verify that the mismatch error shows and the text is as following:
_“This network doesn’t match its associated chain ID or name. Many popular tokens use the name [NAME], making it a target for scams. Scammers may trick you into sending them more valuable currency in return. Verify everything before you continue.”_

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**
image

### **After**
image

## **Related issues**

Fixes https://github.com/MetaMask/MetaMask-planning/issues/1495

## **Pre-merge author checklist**

- [X] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [X] I've clearly explained:
  - [X] What problem this PR is solving.
  - [X] How this problem was solved.
  - [X] How reviewers can test my changes.
- [X] I’ve indicated what issue this PR is linked to: Fixes #???
- [X] I’ve included tests if applicable.
- [X] I’ve documented any added code.
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [X] I’ve properly set the pull request status:
  - [X] In case it's not yet "ready for review", I've set it to "draft".
  - [X] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/pages/settings/networks-tab/networks-form/networks-form.js [hunk 1]
```diff
@@ -691,6 +690,12 @@ const NetworksForm = ({
         />
         <FormField
           warning={warnings.ticker?.msg || ''}
+          warningProps={{
+            'data-testid': 'network-form-ticker-warning',
+            style: {
+              color: 'var(--color-warning-default)',
+            },
```
Comment (georgewrmarshall) [inline ui/pages/settings/networks-tab/networks-form/networks-form.js:697]: Would you be able to use the Text color prop instead? 

```suggestion
            color: TextColor.warningDefault
```

**Review by georgewrmarshall — COMMENTED** [review]
Left one comment

File: ui/pages/settings/networks-tab/networks-form/networks-form.js (hunk 1 above)
Comment (vthomas13) [reply ui/pages/settings/networks-tab/networks-form/networks-form.js:697]: image

Normally yes, but the TextColor.warningDefault class is getting overwritten by the deprecated form-field's h6 color of TextColor.textAlternative. I used the inline style to override the FormField default color.

Also I want to change this component to use FormTextField if possible, but I think that should be done in a new PR.

**Review by darkwing — APPROVED** [review]

**Review by garrettbear — APPROVED** [review]
LGTM, saw the comment with Geroge on the style situation and all good with your solution here

**Pull request merged**